src/python/pants_release/release.py

```diff
@@ -710,85 +681,6 @@ def build_fs_util() -> None:
     green(f"Built fs_util at {dest_dir / 'fs_util'}.")
 
 
-# TODO: We should be using `./pants package` and `pex_binary` for this...If Pants is lacking in
-#  capabilities, we should improve Pants. When porting, using `runtime_package_dependencies` to do
-#  the validation.
```

Comment:
:tada:

Reply:
I wrote this todo, so I'm extra psyched that it's now done!

Reply:
To be extremely fair, I don't think we can technically build a "universal" PEX still, but also that's hand wavy anyways